src/ontology/cl.Makefile

```diff
@@ -273,7 +273,7 @@ deploy_release:
 TERM_hematopoietic= CL:0000988
 TERM_eye= UBERON:0000970
 
-SLIM_TEMPLATES= blood_and_immune
+SLIM_TEMPLATES= eye
 SLIM_REPORTS = $(foreach n,$(SLIM_TEMPLATES),$(REPORTDIR)/$(n)_upper_slim.csv)
```

Comment:
```suggestion
SLIM_TEMPLATES= blood_and_immune eye
SLIM_REPORTS = $(foreach n,$(SLIM_TEMPLATES),$(REPORTDIR)/$(n)_upper_slim.csv)
```

Need to add the `eye` slim and not remove the `blood_and_immune`. @aleixpuigb you can approve this change here.